apps/explorer/src/ui/ExpandableList.tsx

```diff
@@ -10,11 +11,18 @@ import { Text } from './Text';
 interface ExpandableListProps {
     items: ReactNode[];
     defaultItemsToShow: number;
+    itemsLabel?: string;
+    scrollableContent?: {
+        maxHeight: string;
+        gap: string;
+    };
 }
 
 export function ExpandableList({
     items,
     defaultItemsToShow,
+    itemsLabel,
+    scrollableContent,
```

Comment:
What is this prop for? Seems weird

Reply:
`ExpandableList` takes in an array of elements, rather than `children`. So we need to pass this props in to set the `gap` for the content inside `ExpandableList`, either that or I will need to set margin for each of the elements when passing in as an array. Problem with this is, I might only want the gaps between the elements so it might require some thing like `last:m-0` or something, which I am not sure if that is a better solution.

Without doing this, I would need to re-write `ExpandableList` and I might not have enough time to do that + updating all the existing usage of it to accept `children` while being able to dynamically check for other props like `defaultItemsToShow` for example.

Last option would be to write a new implementation for it, which I had, but now removed in favor of using `ExpandableList`